dallinger/recruiters.py

```diff
@@ -381,6 +487,68 @@ def _handle_exit_form_submission(self, assignment_id: str, participant_id: str):
         q = _get_queue()
         q.enqueue(worker_function, "AssignmentSubmitted", assignment_id, participant_id)
 
+    def load_service(self, sandbox):
+        return _prolific_service_from_config(sandbox)
+
+    def clean_qualifications(self, experiment_details):
+        requirements = []
+        for requirement in experiment_details["eligibility_requirements"]:
+            cleaned_attributes = []
```

Comment:
Refactor into a function called `clean_attributes`

Reply:
Document this function to explain why the attributes need cleaning

Reply:
What about `clean_qualification_attributes`, `clean_attributes` seem too general to me. I documented why Prolific qualifications need cleaning and I wrote a test for it.

Reply:
Yep good point